### PR TITLE
ConfigMap supervisor configs and launch scripts for k8s

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY
This PR will configmap the following files rather than copying them.  

* /usr/bin/launch_awx.sh
* /usr/bin/launch_awx_task.sh
* /supervisor.conf
* /supervisor_task.conf

##### ISSUE TYPE

 - Feature Pull Request

### CODE REVIEW

**Comment by rooftopcellist** [comment]
I found some issues when testing the installer yesterday, still tinkering on this one.

**Comment by rooftopcellist** [comment]
I just finished testing this with a fresh installation on GKE, services come up and I am able to run a job from the UI.  

Unfortunately, we won't be able to remove these files from the Dockerfile, as that would break local AWX installations.

**Review by shanemcd — APPROVED** [review]
Works for me.

File: installer/roles/kubernetes/templates/configmap.yml.j2 [hunk 1]
```diff
@@ -205,6 +205,9 @@ data:
     USE_X_FORWARDED_PORT = True
 
     AWX_CONTAINER_GROUP_DEFAULT_IMAGE = "{{ container_groups_image }}"
+    {% if candlepin_host is defined %}
+    REDHAT_CANDLEPIN_HOST = "{{ candlepin_host  | default(omit) }}"
+    REDHAT_CANDLEPIN_VERIFY = "{{ candlepin_verify | default(omit) }}"
```
Comment (shanemcd) [inline installer/roles/kubernetes/templates/configmap.yml.j2:209]: Are we missing an `endif` here?

File: installer/roles/kubernetes/templates/launch_awx.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    if [ `id -u` -ge 500 ]; then
+        echo "awx:x:`id -u`:`id -g`:,,,:/var/lib/awx:/bin/bash" >> /tmp/passwd
+        cat /tmp/passwd > /etc/passwd
+        rm /tmp/passwd
+    fi
+
+    source /etc/tower/conf.d/environment.sh
+
+    ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m wait_for -a "host=$DATABASE_HOST port=$DATABASE_PORT" all
+    ANSIBLE_REMOTE_TEMP=/tmp ANSIBLE_LOCAL_TEMP=/tmp ansible -i "127.0.0.1," -c local -v -m wait_for -a "host=$MEMCACHED_HOST port=11211" all
```
Comment (shanemcd) [inline installer/roles/kubernetes/templates/launch_awx.yml.j2:52]: If https://github.com/ansible/awx/pull/6526 lands before this, delete this after rebasing.

File: installer/roles/kubernetes/templates/launch_awx.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
```
Comment (shanemcd) [inline installer/roles/kubernetes/templates/launch_awx.yml.j2:18]: If #6526 lands before this, delete this after rebasing.

File: installer/roles/kubernetes/vars/kubernetes.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+---
+uwsgi_bash: "bash -c"
```
Comment (shanemcd) [inline installer/roles/kubernetes/vars/kubernetes.yml]: Can you configure your editor to ensure newlines on save?

Comment (rooftopcellist) [reply installer/roles/kubernetes/vars/kubernetes.yml]: will do.  I think I have it fixed now.  Please let me know if you see any more and I'll happily clean that up.

File: installer/roles/kubernetes/templates/configmap.yml.j2 (hunk 1 above)
Comment (chrismeyersfsu) [reply installer/roles/kubernetes/templates/configmap.yml.j2:209]: I'll fix.

**Comment by chrismeyersfsu** [comment]
recheck

**Comment by chrismeyersfsu** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

**Pull request merged**